### ISSUE TITLE
pdoc fails with Jinja2>=3.0

### ISSUE THREAD

**Issue by Piphi5** [mitmproxy/pdoc#254]
#### Problem Description
Although I am able to build docs locally on my Windows computer, it doesn't work in the [Github Actions Machines](https://github.com/Piphi5/globe-observer-utils/runs/2561612426?check_suite_focus=true) or locally on WSL (tried on regular python files and packages such as pandas).

I'm sure this is most likely a problem on my end, but I haven't been able to find much documentation on what the issue is and how to resolve it.

All of these systems fail with the following error from Jinja:
`AssertionError: Tried to resolve a name to a reference that was unknown to the frame ('m')
`

A Full Stack Trace can be found [here](https://github.com/Piphi5/globe-observer-utils/runs/2561612426?check_suite_focus=true).

#### Steps to reproduce the behavior:
1. On WSL 1, install pdoc with `python3 -m pip install pdoc`
2. Try to generate docs for a python file or package (e.g. `python3 -m pdoc pandas`)

#### System Information
Paste the output of "pdoc --version" here.
```
pdoc: 6.6.0                                                                                                                                            
Python: 3.9.2                                                                                                                                         
Platform: Linux-4.4.0-19041-Microsoft-x86_64-with-glibc2.31
```

**Comment by mhils** [comment]
Could you post the output of `pip freeze`? It looks like Jinja2 had a release just yesterday (https://pypi.org/project/Jinja2/3.0.0/), that might be it. :)

**Comment by mhils** [comment]
After upgrading Jinja2 I'm hitting the same issue on my system and can reproduce now. The temporary workaround is to `pip install Jinja2<3`, but I'll try to get that fixed ASAP.

**Comment by mhils** [comment]
I believe this is a Jinja2 bug, I've just filed https://github.com/pallets/jinja/issues/1427. Let's see if we can come up with a workaround for pdoc in the meantime. :)

**Comment by mhils** [comment]
I've just merged #256 with a workaround, we'll have a release later today :)

**Comment by Piphi5** [comment]
Sounds good, thank you!
